lua/nvim-treesitter/parsers.lua

```diff
@@ -289,13 +289,11 @@ list.cue = {
 
 list.d = {
   install_info = {
-    url = "https://github.com/CyberShadow/tree-sitter-d",
-    files = { "src/parser.c", "src/scanner.cc" },
-    requires_generate_from_grammar = true,
+    url = "https://github.com/gdamore/tree-sitter-d",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
   },
-  -- Generating grammar takes ~60s
-  experimental = true,
-  maintainers = { "@nawordar" },
+  maintainers = { "@nawordar", "@ljmf00" },
```

Comment:
```suggestion
  maintainers = { "@ljmf00" },
```

We should take maintainer status seriously; if you are listed, you will get pinged anytime there is an issue with the language -- be it a reported issue, a PR to the queries, or a breaking lockfile update -- and are expected to react in a timely manner.

Reply:
Yes, its ok. I contribute a lot to D and nvim is my daily driver. I prefer for someone to directly ping me, because looking at the PRs is not feasible to me, but I have an RSS feed for whenever someone mentions me on github, even without a direct ping, or mentions `tree-sitter-d` or other relevant keywords for the projects I maintain. @gdamore is also my work colleague, so I can possibly cooperate with him if anything is related to the upstream.

Reply:
We need to look into automating that, as it's also not really feasible to manually ping maintainers for the 190+ languages we have...

Reply:
I don't use nvim, or I'd probably have done this myself -- thanks @ljmf00 for doing it.

I'm happy to help out.  Especially with the queries - though I'm not aware of the specific nvim selectors that are relevant.

Reply:
They're documented in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md (but we'll rework them a bit to be more in line with upstream/Helix in the near future).

Reply:
There's a way to automatically add them via CODEOWNERS and also use github-actions with an heuristic to automatically mention possible maintainers, like based on the tree-sitter link or add a PR/issue template with a dropdown with the language and then map the language to a person.

Reply:
CODEOWNERS only works for actual org members, so we'd have to add every (community) maintainer to the team. There are options with github actions, as you're mentioning; we are looking into the different options here (as the list of maintainers itself has to be maintained actively).

Reply:
> They're documented in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md (but we'll rework them a bit to be more in line with upstream/Helix in the near future).

I wrote the helix plugin for D. So when that happens it will be easier here.  Having said that it's possible that I am missing some there too - I will check on it.

Reply:
Done